web/cobrands/highways/assets.js

```diff
@@ -0,0 +1,112 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+var defaults = {
+    http_options: {
+        url: "https://struan.tilma.dev.mysociety.org/mapserver/highways",
+        params: {
+            SERVICE: "WFS",
+            VERSION: "1.1.0",
+            REQUEST: "GetFeature",
+            SRSNAME: "urn:ogc:def:crs:EPSG::3857"
+        }
+    },
+    format_class: OpenLayers.Format.GML.v3.MultiCurveFix,
+    asset_type: 'area',
+    max_resolution: 4.777314267158508,
+    min_resolution: 0.5971642833948135,
+    asset_id_field: 'CENTRAL_AS',
+    geometryName: 'msGeometry',
+    srsName: "EPSG:3857",
+    strategy_class: OpenLayers.Strategy.FixMyStreet
+};
+
+var highways_stylemap = new OpenLayers.StyleMap({
+    'default': new OpenLayers.Style({
+        fill: false,
+        stroke: false,
+    })
+});
+
+fixmystreet.assets.add($.extend(true, {}, defaults, {
+    http_options: {
+        params: {
+            TYPENAME: "Highways"
+        }
+    },
+    stylemap: highways_stylemap,
+    always_visible: true,
+
+    non_interactive: true,
+    asset_type: 'area',
+    road: true,
+    all_categories: true,
+    nearest_radius: 15,
+    actions: {
+        found: function(layer, feature) {
+            // this is to stop this firing when we change category
+            if ( fixmystreet.body_overrides.highways_override ) {
+                // but if we've changed location then we want to reset things
+                var lat = $('#fixmystreet\\.latitude').val(),
+                    lon = $('#fixmystreet\\.longitude').val();
+                if ( lat == fixmystreet.body_overrides.location.latitude &&
+                     lon == fixmystreet.body_overrides.location.longitude ) {
+                    return;
+                }
+                fixmystreet.body_overrides.highways_override = false;
+            }
+            $('#highways').remove();
+            if ( !fixmystreet.assets.selectedFeature() ) {
+                fixmystreet.body_overrides.only_send('Highways England');
+                add_highways_warning(feature.attributes.ROA_NUMBER);
+            }
+        },
+        not_found: function(layer) {
+            fixmystreet.body_overrides.highways_override = false;
+            fixmystreet.body_overrides.remove_only_send();
+            $('#highways').remove();
+        }
+    }
+}));
+
+function add_highways_warning(road_name) {
+  var $warning = $('<div class="box-warning" id="highways"><p>It looks like you clicked on the ' + road_name + ' which is managed by <strong>Highways England</strong>.<p></div>');
+    $('<a>')
+        .attr('href', '#')
+        .attr('id', 'js-not-highways')
+        .html('Send to <strong>' + fixmystreet.bodies.join('</strong> ' + translation_strings.or + ' <strong>') + '</strong> instead.')
```

Comment:
I think the wording here (and related) needs to help the user know whether they should click it or not. I think making it a radio question would make it clearer perhaps? It could say something like:
"You’ve clicked on the A46, managed by Highways England.
Does your report actually concern something on this road, or somewhere else (e.g. a road crossing it)?
(x) On the A46 ( ) Somewhere else"